portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Welcome to Evie's Portfolio!</h1>
+      <p style = "text-align:center"><img src = "/images/ProfileImage.jpg" alt = "Evie's profile picture"></p>
```

Comment:
**nit**: consider using an id or class for this paragraph element and then styling it within the style.css file.

(in case you are unfamiliar, **nit** is just industry code review jargon for "nitpick", a comment on something that is not very important/the reviewer considers optional for you to fix)